js/src/hooks/types.js

```diff
@@ -0,0 +1,12 @@
+/**
```

Comment:
💅  Since is there not a consensus yet about this `types.js` standard and `ImageMedia` is only being used in `useCroppedImageSelector.js` I would add it there directly deleting this file.

Reply:
The reason is similar to https://github.com/woocommerce/google-listings-and-ads/pull/1805#discussion_r1044303992. I would like to advocate that the type definitions of hooks be centralized in this file.

Reply:
Yes. I understand the idea. But I would suggest to have a team consensus. Maybe a P2 or an agenda question to be all in the same page about the new proposal.

Reply:
Surely it's nicer to have a team consensus for this, and the discussion of this convention perhaps could be covered in #195.